fsd_utils/eoi/evaluate_eoi_response.py

```diff
@@ -0,0 +1,91 @@
+from enum import IntEnum
+
+
+class Eoi_Decision(IntEnum):
+    PASS = 0
+    PASS_WITH_CAVEATS = 1
+    FAIL = 2
+
+
+VALID_OPERATORS = ["<", "<=", "==", ">=", ">"]
+
+
+def _evaluate_with_supplied_operators(
+    conditions_to_evaluate: list, supplied_answer: any
+) -> tuple[Eoi_Decision, list]:
+    """Evaluates an expression built from the operator in the schmea, the value to compare, and the supplied answer. Uses the result of the evaluation to return a decision and applicable caveats
+
+    Args:
+        conditions_to_evaluate (list): List of conditions from schema
+        supplied_answer (any): Answer supplied to this question
+
+    Raises:
+        ValueError: If the operator from the schema is not supported
+
+    Returns:
+        tuple[Eoi_Decision, list]: Tuple of the decision and the caveats (if there are any)
+    """
+    decision = Eoi_Decision.PASS
+    caveats = []
+    for ec in conditions_to_evaluate:
+        # validate supplied operator
+        if not ec["operator"] in VALID_OPERATORS:
```

Comment:
More pythonic and removes the negation:

```suggestion
        if ec["operator"] not in VALID_OPERATORS:
```